js/app/src/Blocks.svelte

```diff
@@ -638,6 +638,7 @@
 			}
 		});
 
+		if (render_complete) return;
```

Comment:
event listener should only be attached once, otherwise every reload doubles up the listeners.

Reply:
I don't follow what this line is doing